community/pdfcpu/description

```diff
@@ -0,0 +1 @@
+a pdf processor written in go
```

Comment:
The `description` file is only used in some personal repos (There's an extension called `kiss-describe`) but not supposed to be present in "official" repos.